etc/daemons/arbiterd.ini

```diff
@@ -7,6 +7,10 @@ workdir=/usr/local/var/run/alignak
 logdir=/usr/local/var/log/alignak
 etcdir=/usr/local/etc/alignak
 
+#-- Note that those variables:
+# 1/ are used in this file as %(workdir)s
+# 2/ are automatically updated during the Alignak installation process (eg. python setup.py install)
```

Comment:
No, no and no, you say since 2 days install with pip and here you say use `python setup.py install` :D